### PR TITLE
MODFEE-204 - Source for ManualBlock deleting

### PR DESCRIPTION
**Pull request by obozhko-folio**
[MODFEE-204](https://issues.folio.org/browse/MODFEE-204) - Source for ManualBlock deleting

## Purpose
Log record for manual block deleting should contain source with user name who deleted this manual block.

## Approach
- [APPROACH 1](https://wiki.folio.org/pages/viewpage.action?spaceKey=FOLIJET&title=User+name+during+manual+block+deleting)
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/rest/service/EventPublisher.java [hunk 1]
```diff
@@ -61,6 +63,7 @@ private Event createEvent(EventType eventType, String payload) {
       .withEventMetadata(new EventMetadata()
         .withPublishedBy(PubSubClientUtils.getModuleId())
         .withTenantId(tenantId)
-        .withEventTTL(1));
+        .withEventTTL(1)
+        .withCreatedBy(okapiHeaders.get("source")));
```
Comment (khandramai) [inline src/main/java/org/folio/rest/service/EventPublisher.java:67]: source value should be located in the payload not in the event metadata etc.

Comment (obozhko-folio) [reply src/main/java/org/folio/rest/service/EventPublisher.java]: I fixed the code, thank you.

**Review by siarhei-charniak — APPROVED** [review]

**Comment by IlliaDaliek** [comment]
> INFO: lint-raml: Using lint-raml version: 1.4.5
> INFO: lint-raml: Investigating and determining configuration: mod-feesfines/ramls
> INFO: lint-raml-schema: Assessing schema files (https://dev.folio.org/guides/describe-schema/):
> INFO: lint-raml-schema: Found 80 JSON schema files under directory './ramls'
> ERROR: lint-raml-schema: inventory/holdingsrecord.json: Missing "description" for: acquisitionFormat, acquisitionMethod, bareHoldingsItems, digitizationPolicy, electronicAccess, entries, holdingsInstance, holdingsItems, holdingsStatements, holdingsStatementsForIndexes, holdingsStatementsForSupplements, instanceId, permanentLocation, receivingHistory
> ERROR: lint-raml-schema: inventory/item.json: Missing "description" for: itemNoteType, permanentLocation, temporaryLocation
> ERROR: lint-raml-schema: inventory/location.json: Missing "description" for: campus, institution, library
> ERROR: lint-raml-schema: kv-configuration.json: Missing top-level "description".
> ERROR: lint-raml-schema: kv-configuration.json: Missing "description" for: code, configName, default, description, enabled, module, userId, value
> ERROR: lint-raml-schema: kv-configurations.json: Missing top-level "description".
> ERROR: lint-raml-schema: kv-configurations.json: Missing "description" for: configs
> ERROR: lint-raml-schema: loan-policy.json: Missing top-level "description".
> ERROR: lint-raml-schema: loan-policy.json: Missing "description" for: description, holds, loanable, loansPolicy, name, pages, recalls, renewalsPolicy, requestManagement
> INFO: lint-raml: Not assessing RAML/Schema or examples against schema, as per option '--json-only'.
> ERROR: lint-raml: There were processing errors. See list above.
> Note: When those ERRORs are fixed, then any INFO and WARNING messages will be gone too. See Jenkins "Artifacts" tab for LintRamlSchemaReport.

@obozhko-folio, could you please fix these raml errors in order to exclude this error message from all further PR's.

**Comment by obozhko-folio** [comment]
@IlliaDaliek , it is only a draft PR with PoC and should be deleted in my opinion, thereby it does not affect further PRs

**Review by khandramai — APPROVED** [review]

**Pull request merged**